wire/modules/Process/ProcessModule/ProcessModule.module

```diff
@@ -206,6 +215,11 @@ class ProcessModule extends Process {
 				// check that it can be installed NOW (i.e. all dependencies met)
 				if(!$this->wire('modules')->isInstallable($moduleName, true)) continue; 
 			}
+
+			if($translated) {
+				// exclude modules without translations present
+				if(!$info['languages'] || !is_array($info['languages']) || !count($info['languages'])) continue;
```

Comment:
Yeah, had that though yesterday evening as well. Might just use !empty(), which coverts the isset as well as count() issue.